src/modules/launcher/Plugins/Microsoft.Plugin.WindowWalker/Properties/Resources.resx

```diff
@@ -190,4 +190,16 @@
   <data name="wox_plugin_windowwalker_SettingKillProcessTree" xml:space="preserve">
     <value>Kill process and its child processes</value>
   </data>
+  <data name="wox_plugin_windowwalker_SettingExplorerSettingInfo_Description" xml:space="preserve">
+    <value>This message is only shown, when searching with direct activation command.</value>
+  </data>
+  <data name="wox_plugin_windowwalker_SettingKillProcessTree_Description" xml:space="preserve">
+    <value>Be careful when activating this.</value>
```

Comment:
Be careful why? We should explain what can go wrong when warning people :)

Reply:
What about "Killing the whole process tree can lead to unexpected unsafe application crashes."